lib/game_logic.rb

```diff
@@ -0,0 +1,67 @@
+class Board
+  @board = [
+    '', '', '',
+    '', '', '',
+    '', '', ''
+  ]
+
+  def self.print_board(selected_cell, user_turn)
+    @board[selected_cell - 1] = user_turn
+
+    puts ''
+    print @board[0..2]
+    puts ''
+    print @board[3..5]
+    puts ''
+    print @board[6..8]
+  end
+end
```

Comment:
There should be no puts or print outside the main.rb file.
![only_main_print](https://user-images.githubusercontent.com/32361963/101890409-b5c88680-3bb1-11eb-9993-4fdef74da2aa.png)